NorwegianRefugeeCouncil/core
# Task/core24 200 create prm component package

Jira ticket: JIRA-CORE24-200


## What have you implemented?
* Add prm components lib with storybook
* Add stubs for prm context
* Add stubs for entity detail page
* Add stubs for prm client/participant client
* Add routing for entity detail page
* Refactor routing to use `createBrowserRouter` with children

## How to test it?
* Run storybook and see component stub rendered
* Run frontend and try out new routes

## Extra/next steps
* Setup component library (likely chakra ui) with the prm component lib
